Fix/Remove the update column under team roster

On the roster view under the teams tab in the league dashboard, the update column needs to be removed from the array that is used to render the columns. We no longer want to update players through the teams.

I get two errors when running npm test before making any modifications
 
    ...league-for-good\client\src\components\dashboard\helper\tableTemplate\__original              Table.jsx
    98:24  error  'onSort' is missing in props validation           react/prop-types
    100:32  error  'sortColumnIndex' is missing in props validation  react/prop-types

--- a/client/src/components/dashboard/teams/team_roster_list/rosterData.js
+++ b/client/src/components/dashboard/teams/team_roster_list/rosterData.js
@@ -31,10 +31,6 @@ const colData = [
 	{
 		label: 'Phone',
 		cellProp: 'phoneNum'
-	},
-	{
-		label: 'Update',
-		cellProp: 'link'
 	}
 ];
 

